src/main/scala/com/wavesplatform/Application.scala

```diff
@@ -20,6 +20,7 @@ import scorex.utils.ScorexLogging
 import scorex.waves.http.{DebugApiRoute, WavesApiRoute}
 import scorex.waves.transaction.WavesTransactionModule
 
+import scala.Option.option2Iterable
```

Comment:
Explicitly importing this one is unnecessary — implicit conversions are searched for in a companion object by default.